### ISSUE TITLE
Failure to rename and find files/par in weirdly obfuscated nzb

### ISSUE THREAD

**Issue by fryfrog** [sabnzbd/sabnzbd#1262]
NZBGet processes this nzb just fine as seen in https://paste.ee/p/KzVBE. It looks like the correct names are contained in some of the smaller files along w/ a checksum. I honestly don't get from the logs how nzbget is getting it right... but it is. When downloaded via sabnzbd, it is just a handful of weirdly named files.

```
165099-0.0    165099-1.0    165099-1.1.1  165099-2.0    165099-2.4  165099-2.7  165099-3.0    165099-3.10   165099-3.1.10  165099-3.1.13  165099-3.1.3  165099-3.1.5  165099-3.1.8  165099-3.2.0  165099-3.5  165099-3.8
165099-0.1.0  165099-1.1    165099-1.2.0  165099-2.1.0  165099-2.5  165099-2.8  165099-3.1    165099-3.1.1  165099-3.1.11  165099-3.1.2   165099-3.13   165099-3.1.6  165099-3.1.9  165099-3.3    165099-3.6  165099-3.9
165099-0.2.0  165099-1.1.0  165099-1.2.1  165099-2.3    165099-2.6  165099-2.9  165099-3.1.0  165099-3.11   165099-3.1.12  165099-3.12    165099-3.1.4  165099-3.1.7  165099-3.2    165099-3.4    165099-3.7
```

But for example, the file `165099-3.6` contains:
```
the.simpsons.s16e21.1080p.bluray.x264-rovers.r00 f33a5300
the.simpsons.s16e21.1080p.bluray.x264-rovers.r01 3835cc58
the.simpsons.s16e21.1080p.bluray.x264-rovers.r02 eb9d1b13
the.simpsons.s16e21.1080p.bluray.x264-rovers.r03 aa8cd45d
the.simpsons.s16e21.1080p.bluray.x264-rovers.r04 4b0eb82f
the.simpsons.s16e21.1080p.bluray.x264-rovers.r05 8a623f7f
the.simpsons.s16e21.1080p.bluray.x264-rovers.r06 be5a4225
the.simpsons.s16e21.1080p.bluray.x264-rovers.r07 c7c62043
the.simpsons.s16e21.1080p.bluray.x264-rovers.r08 26ed2a3b
the.simpsons.s16e21.1080p.bluray.x264-rovers.r09 3b5653cf
the.simpsons.s16e21.1080p.bluray.x264-rovers.r10 aad2f093
the.simpsons.s16e21.1080p.bluray.x264-rovers.rar 9b92b362
```

[The.Simpsons.S16E20.1080p.BluRay.x264-ROVERS.nzb.zip](https://github.com/sabnzbd/sabnzbd/files/3009647/The.Simpsons.S16E20.1080p.BluRay.x264-ROVERS.nzb.zip)

**Comment by PintjesBier** [comment]
Same thing here!

**Comment by fryfrog** [comment]
To be clear, we're troubleshooting this together on Discord. @PintjesBier is the one w/ the problem, we've just been trying to figure it out. :)

**Comment by Safihre** [comment]
Nzbget uses a special way to extract the Rar volume number in case par2 is missing. I hope to implement this soon.

**Comment by s0mm3rb** [comment]
I've noticed this too a couple of times
I just thought the files had been included multiple times in the nzb
sab showed double the size and all parts showed up twice with ".1" added at the end or ".2, .3, ..."
DL, unpack and move however worked fine
just the .1, .2, ... files kept remaning and weren't deleted

**Comment by Safihre** [comment]
Do people maybe have more example NZB's? @sanderjo made a Python version of the fix, now want to implement it!

**Comment by sanderjo** [comment]
OK, some notes and analysis of this post.

I wrote a bash script to do analysis per file:

```
#!/bin/bash
for f in *
do 
	echo -n -e "File $f:\t"
	# check if rar file:
	unrar l $f | grep "not RAR" 2>&1 > /dev/null
	ISRARFILE=$?
	if [ "$ISRARFILE" -eq "0" ]; then
		#echo "Not a rar"
		file $f
	else
		echo -n "RAR-file. Volume "
		unrar l $f | grep volume | tail -1 | awk '{ print $3 }' | tr -d "\n"
		unrar l $f | grep -A2 Attributes | tail -1 | tr -d "\n"
		echo ""	
	fi

done
```

**Comment by sanderjo** [comment]
Result:
```
$ ./inspect-all-files.sh 
File 165099-0.0:	165099-0.0: Matroska data
File 165099-0.1.0:	165099-0.1.0: Matroska data
File 165099-0.2.0:	165099-0.2.0: Matroska data
File 165099-1.0:	165099-1.0: ASCII text, with CRLF line terminators
File 165099-1.1:	RAR-file. Volume     ..A....    761434  2013-11-21 10:58  the.simpsons.s16e19.1080p.bluray.x264-rovers.rar
File 165099-1.1.0:	165099-1.1.0: ASCII text, with CRLF line terminators
File 165099-1.1.1:	RAR-file. Volume     ..A....    711061  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.rar
File 165099-1.2.0:	165099-1.2.0: ASCII text, with CRLF line terminators
File 165099-1.2.1:	RAR-file. Volume     ..A....    839707  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.rar
File 165099-2.0:	165099-2.0: JPEG image data, baseline, precision 8, 3264x2448, components 3
File 165099-2.1.0:	165099-2.1.0: JPEG image data, baseline, precision 8, 3264x2448, components 3
File 165099-2.3:	RAR-file. Volume 6    ..A.... 1167834162  2013-11-21 10:58  the.simpsons.s16e19.1080p.bluray.x264-rovers.mkv
File 165099-2.4:	RAR-file. Volume 3    ..A.... 1167834162  2013-11-21 10:58  the.simpsons.s16e19.1080p.bluray.x264-rovers.mkv
File 165099-2.5:	RAR-file. Volume 4    ..A.... 1167834162  2013-11-21 10:58  the.simpsons.s16e19.1080p.bluray.x264-rovers.mkv
File 165099-2.6:	RAR-file. Volume 12    ..A.... 1167834162  2013-11-21 10:58  the.simpsons.s16e19.1080p.bluray.x264-rovers.mkv
File 165099-2.7:	RAR-file. Volume 5    ..A.... 1167834162  2013-11-21 10:58  the.simpsons.s16e19.1080p.bluray.x264-rovers.mkv
File 165099-2.8:	RAR-file. Volume 10    ..A.... 1167834162  2013-11-21 10:58  the.simpsons.s16e19.1080p.bluray.x264-rovers.mkv
File 165099-2.9:	RAR-file. Volume 7    ..A.... 1167834162  2013-11-21 10:58  the.simpsons.s16e19.1080p.bluray.x264-rovers.mkv
File 165099-3.0:	165099-3.0: JPEG image data, baseline, precision 8, 3264x2448, components 3
File 165099-3.1:	RAR-file. Volume 6    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.1.0:	RAR-file. Volume 8    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.10:	RAR-file. Volume 1    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.1.1:	RAR-file. Volume 9    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.11:	RAR-file. Volume 7    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.1.10:	RAR-file. Volume 3    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.1.11:	RAR-file. Volume 1    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.1.12:	RAR-file. Volume 7    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.1.13:	RAR-file. Volume 5    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.1.2:	RAR-file. Volume 4    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.12:	165099-3.12: ASCII text, with CRLF line terminators
File 165099-3.1.3:	RAR-file. Volume 2    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.13:	RAR-file. Volume 9    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.1.4:	RAR-file. Volume 10    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.1.5:	RAR-file. Volume 6    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.1.6:	RAR-file. Volume 12    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.1.7:	RAR-file. Volume 5    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.1.8:	RAR-file. Volume 8    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.1.9:	RAR-file. Volume 12    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.2:	165099-3.2: Non-ISO extended-ASCII text
File 165099-3.2.0:	RAR-file. Volume 11    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.3:	RAR-file. Volume 2    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.4:	RAR-file. Volume 4    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.5:	RAR-file. Volume 3    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.6:	165099-3.6: ASCII text, with CRLF line terminators
File 165099-3.7:	165099-3.7: Non-ISO extended-ASCII text
File 165099-3.8:	RAR-file. Volume 11    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.9:	RAR-file. Volume 10    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File inspect-all-files.sh:	inspect-all-files.sh: Bourne-Again shell script, ASCII text executable

```

**Comment by sanderjo** [comment]
Now let's look at the RAR files only:
```
$ ./inspect-all-files.sh  | grep RAR | awk '{ print $NF, $0 }' | sort | cut -d' ' -f2-
File 165099-2.3:	RAR-file. Volume 6    ..A.... 1167834162  2013-11-21 10:58  the.simpsons.s16e19.1080p.bluray.x264-rovers.mkv
File 165099-2.4:	RAR-file. Volume 3    ..A.... 1167834162  2013-11-21 10:58  the.simpsons.s16e19.1080p.bluray.x264-rovers.mkv
File 165099-2.5:	RAR-file. Volume 4    ..A.... 1167834162  2013-11-21 10:58  the.simpsons.s16e19.1080p.bluray.x264-rovers.mkv
File 165099-2.6:	RAR-file. Volume 12    ..A.... 1167834162  2013-11-21 10:58  the.simpsons.s16e19.1080p.bluray.x264-rovers.mkv
File 165099-2.7:	RAR-file. Volume 5    ..A.... 1167834162  2013-11-21 10:58  the.simpsons.s16e19.1080p.bluray.x264-rovers.mkv
File 165099-2.8:	RAR-file. Volume 10    ..A.... 1167834162  2013-11-21 10:58  the.simpsons.s16e19.1080p.bluray.x264-rovers.mkv
File 165099-2.9:	RAR-file. Volume 7    ..A.... 1167834162  2013-11-21 10:58  the.simpsons.s16e19.1080p.bluray.x264-rovers.mkv
File 165099-1.1:	RAR-file. Volume     ..A....    761434  2013-11-21 10:58  the.simpsons.s16e19.1080p.bluray.x264-rovers.rar
File 165099-3.10:	RAR-file. Volume 1    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.1.0:	RAR-file. Volume 8    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.11:	RAR-file. Volume 7    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.13:	RAR-file. Volume 9    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.1.6:	RAR-file. Volume 12    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.1.7:	RAR-file. Volume 5    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.1:	RAR-file. Volume 6    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.3:	RAR-file. Volume 2    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.4:	RAR-file. Volume 4    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.5:	RAR-file. Volume 3    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.8:	RAR-file. Volume 11    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.9:	RAR-file. Volume 10    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-1.1.1:	RAR-file. Volume     ..A....    711061  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.rar
File 165099-3.1.10:	RAR-file. Volume 3    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.1.11:	RAR-file. Volume 1    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.1.12:	RAR-file. Volume 7    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.1.13:	RAR-file. Volume 5    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.1.1:	RAR-file. Volume 9    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.1.2:	RAR-file. Volume 4    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.1.3:	RAR-file. Volume 2    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.1.4:	RAR-file. Volume 10    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.1.5:	RAR-file. Volume 6    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.1.8:	RAR-file. Volume 8    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.1.9:	RAR-file. Volume 12    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.2.0:	RAR-file. Volume 11    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-1.2.1:	RAR-file. Volume     ..A....    839707  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.rar

```

**Comment by sanderjo** [comment]
This one looks good: all rar volumes are there:

```
$ ./inspect-all-files.sh  | grep RAR | awk '{ print $NF, $0 }' | sort | cut -d' ' -f2- | grep s16e20 | sort -n -k 5,5
File 165099-1.1.1:	RAR-file. Volume     ..A....    711061  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.rar
File 165099-3.10:	RAR-file. Volume 1    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.3:	RAR-file. Volume 2    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.5:	RAR-file. Volume 3    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.4:	RAR-file. Volume 4    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.1.7:	RAR-file. Volume 5    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.1:	RAR-file. Volume 6    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.11:	RAR-file. Volume 7    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.1.0:	RAR-file. Volume 8    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.13:	RAR-file. Volume 9    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.9:	RAR-file. Volume 10    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.8:	RAR-file. Volume 11    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
File 165099-3.1.6:	RAR-file. Volume 12    ..A.... 1167786443  2013-11-21 11:16  the.simpsons.s16e20.1080p.bluray.x264-rovers.mkv
```
this one good too:
```
$ ./inspect-all-files.sh  | grep RAR | awk '{ print $NF, $0 }' | sort | cut -d' ' -f2- | grep s16e21 | sort -n -k 5,5
File 165099-1.2.1:	RAR-file. Volume     ..A....    839707  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.rar
File 165099-3.1.11:	RAR-file. Volume 1    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.1.3:	RAR-file. Volume 2    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.1.10:	RAR-file. Volume 3    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.1.2:	RAR-file. Volume 4    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.1.13:	RAR-file. Volume 5    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.1.5:	RAR-file. Volume 6    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.1.12:	RAR-file. Volume 7    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.1.8:	RAR-file. Volume 8    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.1.1:	RAR-file. Volume 9    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.1.4:	RAR-file. Volume 10    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.2.0:	RAR-file. Volume 11    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv
File 165099-3.1.9:	RAR-file. Volume 12    ..A.... 1167899972  2013-11-21 11:51  the.simpsons.s16e21.1080p.bluray.x264-rovers.mkv

```

And this one looks to have missing volume:
```
$ ./inspect-all-files.sh  | grep RAR | awk '{ print $NF, $0 }' | sort | cut -d' ' -f2- | grep s16e19 | sort -n -k 5,5
File 165099-1.1:	RAR-file. Volume     ..A....    761434  2013-11-21 10:58  the.simpsons.s16e19.1080p.bluray.x264-rovers.rar
File 165099-2.4:	RAR-file. Volume 3    ..A.... 1167834162  2013-11-21 10:58  the.simpsons.s16e19.1080p.bluray.x264-rovers.mkv
File 165099-2.5:	RAR-file. Volume 4    ..A.... 1167834162  2013-11-21 10:58  the.simpsons.s16e19.1080p.bluray.x264-rovers.mkv
File 165099-2.7:	RAR-file. Volume 5    ..A.... 1167834162  2013-11-21 10:58  the.simpsons.s16e19.1080p.bluray.x264-rovers.mkv
File 165099-2.3:	RAR-file. Volume 6    ..A.... 1167834162  2013-11-21 10:58  the.simpsons.s16e19.1080p.bluray.x264-rovers.mkv
File 165099-2.9:	RAR-file. Volume 7    ..A.... 1167834162  2013-11-21 10:58  the.simpsons.s16e19.1080p.bluray.x264-rovers.mkv
File 165099-2.8:	RAR-file. Volume 10    ..A.... 1167834162  2013-11-21 10:58  the.simpsons.s16e19.1080p.bluray.x264-rovers.mkv
File 165099-2.6:	RAR-file. Volume 12    ..A.... 1167834162  2013-11-21 10:58  the.simpsons.s16e19.1080p.bluray.x264-rovers.mkv

```
Weird. Or my method is incorrect.

**Comment by sanderjo** [comment]
FYI: https://github.com/sabnzbd/sabnzbd/issues/1331 contains my analysis of my own (created) obfuscated, mixed-rar set.

**Comment by PintjesBier** [comment]
> 
> 
> Do people maybe have more example NZB's? @sanderjo made a Python version of the fix, now want to implement it!

I don't, sorry! Never encountered another file like this...

**Comment by Safihre** [comment]
Fixed in the new SABnzbd 3.0.0 release 🛩️
